Cargo.toml

```diff
@@ -13,26 +13,17 @@ readme = "README.md"
 [dependencies]
 anyhow = "1.0.32"
 base64 = "0.12.3"
+bytes = "1.0"
 ctrlc = {version = "3.1.6", features = ["termination"] }
+chrono = "0.4.18"
 fern = "0.6.0"
 log = "0.4.11"
-rayon = "1.4.0"
-scopeguard = "1.1.0"
-serde = "1.0.116"
+reqwest = { version = "0.11.3", features = [ "blocking"] }
+rpki = { version = "0.11.1-dev", features = [ "rrdp", "serde" ], git = "https://github.com/NLnetLabs/rpki-rs.git", branch = "main" }
```

Comment:
Is rpki-rs 0.11.1 not yet ready for release?

Reply:
It will be before releasing this as 0.2, but since this is a PR to the dev branch and not yet the actual release I think this should not be a showstopper.